Microsoft.PowerShell.WhatsNew/relnotes/What-s-New-in-PowerShell-70.md

```diff
@@ -1,10 +1,10 @@
 ---
 description: New features and changes released in PowerShell 7.0
-ms.date: 10/22/2021
-title: What's New in PowerShell 7.0
+ms.date: 11/16/2022
```

Comment:
Should this be today 11/29?

Reply:
No, that is when the file was last updated.